module/cpf.sh

```diff
@@ -26,17 +29,22 @@ declare -A COLORS=(
 )
 
 #. cpf:module -={
-function ::cpf:module_is_modified() {
+function ::cpf:module_is_unchanged() {
```

Comment:
Also you changed the function name to "is_unchanged", but then the variable receiving this answer is called "amended".  That is very confusing.